config/routes.rb

```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  root 'static_pages#home'
+  get 'static_pages/home'
+  get '/signup', to: 'users#new'
+  resources :users
```

Comment:
現状 `new` と `showしかないので`
```suggestion
  resources :users, only: %i(new show)
```